### PR TITLE
Returns the identifier if the type is set to uuid.

### PR DESCRIPTION
**Pull request by Batisska**

### CODE REVIEW

**Comment by Batisska** [comment]
@santigarcor

File: src/Helper.php [hunk 1]
```diff
@@ -12,8 +13,9 @@ class Helper
     /**
      * Gets the id from an array, object or integer.
      *
-     * @param  mixed  $object
-     * @param  string  $type
+     * @param mixed  $object
+     * @param string $type
+     *
```
Comment (KKSzymanowski) [inline src/Helper.php:18]: If this PR is still relevant, please revert all unrelated changes like this reformatting.

**Comment by filippotoso** [comment]
I'm going to close this PR and move the improvements in a separate PR.
